README.md

```diff
@@ -30,10 +30,13 @@ Then you can see suggestions in your IDE or you can run checks manually:
 ```bash
 dart analyze;
 dart run dart_code_metrics:metrics analyze lib;
+dart run dart_code_metrics:metrics analyze test;
 dart run dart_code_metrics:metrics check-unused-files lib;
 dart run dart_code_metrics:metrics check-unused-l10n lib;
```

Comment:
```suggestion
dart run dart_code_metrics:metrics analyze lib test;
dart run dart_code_metrics:metrics check-unused-files lib test;
dart run dart_code_metrics:metrics check-unused-l10n lib test;
```

Reply:
Maybe we also should mention that `example` folder should be checked explicitly. 
Not sure we want to include it here -- it'll probably break the command for any project that doesn't have an example. But probably worth mentioning as a note below these commands.

Reply:
Great suggestion! Thanks!

Reply:
I think covering `example` maybe too much and will add more complexity to the docs? As it's a separate app it's generally makes sense that it's not recursive. Another case could be monorepo - we'll probably adjust for that case soon too? ;)